### PR TITLE
docs: Correcting various grammar and spelling errors.

### PR DESCRIPTION
**Pull request by jnh5y**
### Description 
Fixes various typos.

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: docs/developer-guide/ksqldb-rest-api/query-endpoint.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Response JSON Object:
 
 - **header** (object): Information about the result.
     - **header.queryId**: (string): the unique id of the query. This can be useful when debugging. 
-    For example, when looking in the logs or processing log for errors or issues.
+    For example, when looking in the logs or processing logs for errors or issues.
```
Comment (colinhicks) [inline docs/developer-guide/ksqldb-rest-api/query-endpoint.md:40]: In this case _processing log_ refers to the singular `KSQL_PROCESSING_LOG`. I suggest leaving as-is or updating the change to use the explicit `KSQL_PROCESSING_LOG` term.

File: docs/how-to-guides/update-a-running-persistent-query.md [hunk 2]
```diff
@@ -39,14 +39,14 @@ ksqlDB provides two mechanisms to change a query that is already running:
...
-   To accomplish this, users you first issue a `TERMINATE <query_id>;` and a
+   To accomplish this, you first issue a `TERMINATE <query_id>;` and a
    `DROP <source>` before creating the query again.
 
 ## Understanding upgrades
 
 Obviously, it would be preferable to always perform an in-place upgrade
 when you change a query. But because of how streaming programs are constructed,
-this isn't not always possible to do that.
+it isn't not always possible to do thatso.
```
Comment (colinhicks) [inline docs/how-to-guides/update-a-running-persistent-query.md:49]:
```suggestion
this is not always possible.
```

**Review by colinhicks — APPROVED** [review]
Thank you!

File: docs/developer-guide/ksqldb-rest-api/query-endpoint.md (hunk 1 above)
Comment (spena) [reply docs/developer-guide/ksqldb-rest-api/query-endpoint.md:40]: It'd be good to specify `or the ksql processing log ...` to make it more explicit.

**Review by spena — APPROVED** [review]

File: docs/developer-guide/ksqldb-rest-api/query-endpoint.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Response JSON Object:
 
 - **header** (object): Information about the result.
     - **header.queryId**: (string): the unique id of the query. This can be useful when debugging. 
-    For example, when looking in the logs or processing log for errors or issues.
+    For example, when looking in the logs or the ksql processing log for errors or issues.
```
Comment (JimGalasyn) [inline docs/developer-guide/ksqldb-rest-api/query-endpoint.md:40]:
```suggestion
    For example, when looking in the logs or the ksqlDB processing log for errors or issues.

```

**Review by JimGalasyn — APPROVED** [review]
LGTM, with one suggestion. Thanks for the edits!

**Pull request merged**